index.d.ts

```diff
@@ -1523,8 +1524,8 @@ declare namespace Moleculer {
 		show(): this;
 		hide(): this;
 		find(command: string): Vorpal.Command;
-		exec(command: string): Promise<{}>;
-		execSync(command: string): Promise<{}>;
+		exec(command: string): Promise<any>;
```

Comment:
I agree that `{}` is not a good type to use here, but I'm also unclear if `any` is appropriate.  I can't find any reference to `Vorpal` in moleculer so I'm not sure why it's included here at all.  I know `moleculer-repl` is based on the Vorpal library, but I can't find any reference to `exec` or `execSync` in either `moleculer-repl` or Vorpal so I'm totally unsure about this.

`any` is safe so I guess I'm okay with it, but I just don't know why its here at all.